### PR TITLE
made a milestone page under sunbird ed

### PR DESCRIPTION
**Pull request by shivam-2306**
## Project
sunbird ed

### Changes
Added section for content detail page as widget

### CODE REVIEW

File: docs/2023/Sunbird ED/content detail page as widget/index.md
Comment (wasup-yash) [inline docs/2023/Sunbird ED/content detail page as widget/index.md:1]: I think you need to wrap all the links of the contact info in the text [github](https://github.com/shivam-2306) something like this

**Review by wasup-yash — APPROVED** [review]

File: docs/2023/Sunbird ED/content detail page as widget/index.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+---
+
+
+# Author
+Shivam Chaturvedi
+
+# Contact Info
+- Email: chaturvediji2306@gmail.com 
+- Linkedin: https://www.linkedin.com/in/shivam-chaturvedi-035995206/ 
+- Github: https://github.com/shivam-2306
```
Comment (claddyk) [inline docs/2023/Sunbird ED/content detail page as widget/index.md:14]:
```
- [Email](chaturvediji2306@gmail.com)
- [Linkedin](https://www.linkedin.com/in/shivam-chaturvedi-035995206/)
- [Github](https://github.com/shivam-2306)
```

**Review by claddyk — COMMENTED** [review]
Please open a new PR for updating your week 1 & week 2 progress.

File: docs/2023/Sunbird ED/content detail page as widget/index.md (hunk 1 above)
Comment (Shruti3004) [reply docs/2023/Sunbird ED/content detail page as widget/index.md:14]: Please also add mailto:
- [Email](mailto:chaturvediji2306@gmail.com)

Comment (wasup-yash) [reply docs/2023/Sunbird ED/content detail page as widget/index.md:14]: > Please also add mailto: `- [Email](mailto:chaturvediji2306@gmail.com)` @wasup-yash

Right shruti, I think @shivam-2306 needs to update this too

**Review by Shruti3004 — APPROVED** [review]

**Pull request merged**